src/compiler/tsc.ts

```diff
@@ -675,7 +677,22 @@ namespace ts {
             usageText += getParamType(option);
 
             usageColumn.push(usageText);
-            descriptionColumn.push(getDiagnosticText(option.description));
+            let description: string;
+
+            if (option.name === "lib") {
+                description = getDiagnosticText(option.description);
+                const options: string[] = [];
+                const element = (<CommandLineOptionOfListType>option).element;
+                forEachKey(<Map<number | string>>element.type, key => {
+                    options.push(`'${key}'`);
+                });
+                optionsDescriptionMap[description] = options;
+            }
+            else {
+                description = getDiagnosticText(option.description);

```

Comment:
I don't undersatnd. You initialize the exact same way in both branches, and you use `description` in exactly one other place within the `if`. Seems like you could reduce confusion here if you restored the original statement and had a simple `if`.